server/lib/mappings/result-data.json

```diff
@@ -9,7 +9,7 @@
             "format": "dateOptionalTime"
         },
         "@timestamp_original": {
-            "type": "text"
+            "type": "keyword"
```

Comment:
I don't think we want to use `keyword` here for what amounts to random values.  Is the dashboard using this field?  If so, that is probably a bug.

Reply:
This decision was more of a "we don't need full text search support" for this. "keyword" isn't tokenized, which seemed like a win if a small one. I suppose "@timestamp_original": {"type": "keyword", "index": false} might be a step further.